ui/config/webpack.config.js

```diff
@@ -247,8 +254,7 @@ module.exports = function(webpackEnv) {
       // https://twitter.com/wSokra/status/969633336732905474
       // https://medium.com/webpack/webpack-4-code-splitting-chunk-graph-and-the-splitchunks-optimization-be739a861366
       splitChunks: {
-        chunks: 'all',
-        name: false,
```

Comment:
I had to remove this, so that chunks remove their name. This allows us to use `excludeChunks: [...]` below.